docs/Behaviour - Client Side.md

```diff
@@ -2,7 +2,17 @@
 
 ## Active Constraints of Sender
 
-Before making an [IS-05][IS-05] connection, NMOS Controller chooses a Sender and a number of Receivers. Then NMOS Controller MUST `GET /constraints/supported` from the Sender, MUST collect Receiver Capabilities from the Receivers and make such processing of them that they can be utilized for building Active Constraints satisfying all (by default) or some (depending on user preferences) of the Receivers and using Parameter Constraint URNs supported by the Sender. If the Sender supports fewer Parameter Constraint URNs than used in the Receiver Capabilities, the NMOS Controller SHOULD inform the user about it. If the processing of the Receiver Capabilities results in an empty `constraint_sets` of Active Constraints, NMOS Controller SHOULD inform the user about it. After that NMOS Controller MUST `PUT /constraints/active` to the Sender and SHOULD make the IS-05 connections if the Constraints have been applied succesfully to the Sender. After breaking these connections via IS-05, NMOS Controller is RECOMMENDED to `DELETE /constraints/active` of the Sender.
+Before making an [IS-05][IS-05] connection, an NMOS Controller chooses a Sender and a number of Receivers. The NMOS Controller MAY keep the current configuration of the Sender or change its configuration. In case of changing configuration of an active Sender, deactivating such Sender MAY be required and depends on the Sender itself.
```

Comment:
Is the last sentence referring to the possibilities covered in the **Dynamic format changes on Sender** section below? Otherwise the reader naturally asks _how_ it depends on the Sender itself, and how the Controller can determine it.

Reply:
We found this during the call:

https://github.com/AMWA-TV/is-11/blob/309d430f4475aec9307b7dc02d3a2641eaf793f6/APIs/StreamCompatibilityManagementAPI.raml#L123-L125

Reply:
```suggestion
Before making an [IS-05][IS-05] connection, an NMOS Controller chooses a Sender and a number of Receivers. The NMOS Controller MAY keep the current configuration of the Sender or change its configuration.
```

Moving explanation of `423` (Locked) below.